<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab4SensorDisplay</class>
 <widget class="QWidget" name="Tab4SensorDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>401</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>조도 온도 습도 현황</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="pPhoto">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>조도</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pPhotoValue">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="pTemp">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>온도</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pTempValue">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="pHumi">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>습도</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pHumiValue">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
